packages/app/src/systems/Account/machines/accountsMachine.tsx

```diff
@@ -17,74 +17,133 @@ type MachineServices = {
   fetchAccounts: {
     data: Maybe<Account[]>;
   };
+  fetchBalances: {
+    data: Maybe<Account[]>;
+  };
 };
 
 type MachineEvents =
   | { type: "SET_ACCOUNTS"; data: Account[] }
-  | { type: "UPDATE_ACCOUNTS"; data: { account: Account } };
+  | { type: "UPDATE_ACCOUNTS"; data: { account: Account } }
+  | { type: "REFETCH" };
 
-export const accountsMachine = createMachine(
-  {
-    // eslint-disable-next-line @typescript-eslint/consistent-type-imports
-    tsTypes: {} as import("./accountsMachine.typegen").Typegen0,
-    predictableActionArguments: true,
-    id: "(machine)",
-    initial: "fetching",
-    schema: {
-      context: {} as MachineContext,
-      services: {} as MachineServices,
-      events: {} as MachineEvents,
-    },
-    states: {
-      fetching: {
-        invoke: {
-          src: "fetchAccounts",
-          onDone: {
-            target: "done",
-            actions: "assignAccounts",
+export const accountsMachine =
+  /** @xstate-layout N4IgpgJg5mDOIC5QAoC2BDAxgCwJYDswBKAOgDMwAXHAqAQU0wHsBXfS2AYgicJIIBuTANZgSaLHkKkK1KfUat2sBIKaZ0lXLwDaABgC6iUAAcmsXFt7GQAD0QBGAEx6SehwE4ArB71OAHE5eTgDM-gA0IACejh4ALCQAbADsXskeIXohIelecQC++ZESNNLkVKUKzGwcnGAATvVM9SQmADaaZM2o4hilxOVytAzVyqr4QhpW+PpGSCBmFtM29gjODiSZev5xu4leDiGJ7pExCCEeJHFOTh7+Hp6JiZ7+XoXFfVIDspUAQugdfCYODcXhiNSiXqSAjfCryf6A4EqNRTbQzQw2RaWNErRz+EIkTweUKZPI7LyBU6IOKJBKPYIpRKhBx5ApFEAlL4yOG0BHoIEghpNFrtTrdKH9blDfBQPkC5ETdSaNGzTHmbHWearELXQn+ZLJfX7AIskJUhAuDZ6PQGg5pfyJHWJd4cz4w0g8QicADKAFEACoAfToAGEQwB5ACqADl-d61UscVrEABaBzpTYhTx6YIMzIeZLm0JeEjEi45nwhLwXasuznukiesCcABKvoAYgGQwAJBMa-C4hBphKpJ7PfW7HKJfFFvSJK7OPz3AuvfWFdn4JgQOA2etlH7yEZKDh95bJofppwkV4ORIPB6vR0O800zZ3pwOOIGsL7bJ1t37jyMpykip5JqAqy3gSTIOA4egZAWcTONO5ohE4CRBME-g5vqPiBP+0L7uguBtJAYGahBiA5BsLiZPcFxbIW0SIMEVw3Hc75xBSVb+ARkqNmC5EDueaYZs4NzJB+063mOqE+CQBzoXBwQsncbIfIRxBCYOKa0Zm2a5k4jrwUxZwpmhCkGtauzWkc-j4m865AA */
+  createMachine(
+    {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+      tsTypes: {} as import("./accountsMachine.typegen").Typegen0,
+      schema: {
+        context: {} as MachineContext,
+        services: {} as MachineServices,
+        events: {} as MachineEvents,
+      },
+      predictableActionArguments: true,
+      id: "(machine)",
```

Comment:
This should include something like `(machine).<name>` or it just `(machine)`?

Reply:
it's only (machine). same way was done in swayswap and in `signUpMachine` as well